packages/zero-cache/src/server/logging.ts

```diff
@@ -13,12 +13,9 @@ import {OtelLogSink} from './otel-log-sink.ts';
 function createLogSink(config: LogConfig): LogSink {
   const consoleSink =
     config.format === 'json' ? consoleJsonLogSink : consoleLogSink;
-  if (config.logCollector) {
-    const otelSink = new OtelLogSink(config);
-    return new CompositeLogSink([otelSink, consoleSink]);
-  }
-
-  return consoleSink;
+  // Always create OtelLogSink for debugging
+  const otelSink = new OtelLogSink();
```

Comment:
can we check if the otel env vars are set and only create the sink if so? No need to log to OTEL if OTEL is not enabled.